packages/sdks/src/functions/get-block-bindings.ts

```diff
@@ -0,0 +1,21 @@
+import type { BuilderBlock } from '../types/builder-block.js';
+import { evaluate } from './evaluate.js';
+
+export function getBlockBindings(state: any, block: BuilderBlock) {
+  const props: Record<string, any> = {};
+  const bindings = block.bindings;
+  if (bindings) {
+    for (const key in bindings) {
+      if (key.startsWith('component.')) {
+        // ignore
+      } else if (key.startsWith('style.')) {
+        const styleKey = key.substring('style.'.length);
+        const style = props.style || (props.style = {});
+        style[styleKey] = evaluate({ code: bindings[key], state, context: {} });
+      } else {
+        props[key] = evaluate({ code: bindings[key], state, context: {} });
+      }
+    }
+  }
+  return props;
+}
```

Comment:
This code should be evaluating all bindings already: https://github.com/BuilderIO/builder/blob/2d495aba29af37791b283d254f39dc1d5fa6f437/packages/sdks/src/functions/get-processed-block.ts#L8-L33

And it's called due to this flag being `true`: https://github.com/BuilderIO/builder/blob/2d495aba29af37791b283d254f39dc1d5fa6f437/packages/sdks/src/components/render-block/render-block.lite.tsx#L76


so `props.useBlock` should have all bindings evaluated correctly. Why is this function needed? If there's a bug in `evaluateBindings`, the bug should be fixed there directly

Reply:
hey @samijaber so I need some help figuring out what is going on here.

```typescript
    get component(): Nullable<RegisteredComponent> {
      const componentName = getProcessedBlock({
        block: props.block,
        state: props.context.state,
        context: props.context.context,
        shouldEvaluateBindings: false,  // <=== THIS LINE
      }).component?.name;
 ```

So you added `false` at this location, and that is the reason why the bindings are not being processed. Any idea as to why?

Reply:
I have resolved your suggestions. PTAL